docs/docs.html

```diff
@@ -34,6 +34,8 @@
     <!-- sidebar -->
     <div class="screen">
```

Comment:
in Header, if it's only a single icon link, you don't need to nest it inside div, ul,  and then li. This only adds extra code.

Reply:
this has been done for making  a generic header where there may be more links /icons.